exporters/otlp/otlptrace/README.md

```diff
@@ -0,0 +1,28 @@
+# OpenTelemetry-Go OTLP Span Exporter
+
+[![Go Reference](https://pkg.go.dev/badge/go.opentelemetry.io/otel/exporters/otlp/otlptrace.svg)](https://pkg.go.dev/go.opentelemetry.io/otel/exporters/otlp/otlptrace)
+
+OpenTelemetry Protocol (OTLP) Span Exporter.
+
+To constructs a new OTLP trace Exporter, you can follow this 
+ [`example`](https://github.com/open-telemetry/opentelemetry-go/blob/main/exporters/otlp/otlptrace/example_test.go).
```

Comment:
the hyperlink should be relative. Here is an example of how to do it: https://raw.githubusercontent.com/open-telemetry/opentelemetry-go/main/exporters/jaeger/README.md

Personally, instead of putting the info here, I would put the following section after `Installation` section:

```
## Examples

- [example_test.go](example_test.go)
- [../../example/otel-collector](../../example/otel-collector)
```

Reply:
I agree with you .
 Before you wrote your comment, I have just improved the example section according to [https://raw.githubusercontent.com/open-telemetry/opentelemetry-go/main/exporters/jaeger/README.md](https://raw.githubusercontent.com/open-telemetry/opentelemetry-go/main/exporters/jaeger/README.md).
I am so glad that the way I improve the doc is so consistent with yours.